kubernetes/kfserving/handler/v2/mnist_handler.py

```diff
@@ -0,0 +1,28 @@
+from torchvision import transforms
+from ts.torch_handler.image_classifier import ImageClassifier
+import torch
+import captum
+
+class MNISTDigitClassifier(ImageClassifier):
+    """
+    MNISTDigitClassifier handler class. This handler extends class ImageClassifier from image_classifier.py, a
+    default handler. This handler takes an image and returns the number in that image.
+
+    Here method postprocess() has been overridden while others are reused from parent class.
+    """
+
+    image_processing = transforms.Compose([
```

Comment:
I don't see this being used in post process. Is this overriding a method for a derived class?

Reply:
Yes this overrides the derived class.